docs/source/api/plugin/landing-pages.md

```diff
@@ -228,7 +274,7 @@ If you omit this, the Explorer initially loads an example query based on your sc
 
 ###### `variables`
 
-`Record<string, string>`
+`Record<string, string | Record>`
```

Comment:
maybe the right hand side here should just be `unknown` or `any`? i mean you can put numbers and bools and lists in variables...

Reply:
This ticket was for a new config (initialState) recently added to the embedded-explorer repo. These parameters (document, variables and headers) already existed, but yeah they weren't used or passed through for embedded Sandbox.

Reply:
Added this change in #6656